src/axom/quest/tests/quest_point_in_cell_mfem.cpp

```diff
@@ -297,7 +291,7 @@ class PointInCellTest : public ::testing::Test
     // Generate a PointInCell structure over the mesh
     axom::utilities::Timer constructTimer(true);
     // _quest_pic_init_start
-    PointInCellType spatialIndex(m_mesh, GridCell(25).data(), m_allocatorID);
+    PointInCellType spatialIndex(m_mesh, GridCell(25).data(), m_EPS);
```

Comment:
@kanye-quest -- I had to remove the fourth parameter `m_allocatorID` from the `PointInCell` constructor invocations. 
It was running into segfaults on blueos in the `BitSet` constructor (specifically when calling `Array::fill`) since `BitSet` does not (yet?) support modification operations on the device.

Is this something you were planning to continue working on?

Reply:
@kanye-quest and I discussed this offline. I created #757 to track this.